linux-bore/PKGBUILD

```diff
@@ -311,6 +311,9 @@ prepare() {
     echo "Setting config..."
     cp ../config .config
 
+    echo "Setting config..."
```

Comment:
We dont need it doubled.
Please remove the doubled "Settings config..."

Reply:
Sure, I will fix.

Reply:
Done